### ISSUE TITLE
Match multiCase not checking spaces after comma

### ISSUE THREAD

**Issue by HMartusewiczVox** [PHPCSStandards/PHP_CodeSniffer#1369]
PSR says:

<img width="835" height="344" alt="Image" src="https://github.com/user-attachments/assets/b4e78a32-b604-4821-bbcc-8eb4f006c519" />

But the following code raises no issue:

```php
$x = match (1) {
    1,2,3 => 'asdas',
    default => 5
};
```

Demonstrated by:

```plain
[root@b8a7f7560dfb html]# ./vendor/bin/phpcs sniff_match.php 

FILE: /var/www/html/sniff_match.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 8 | ERROR | [x] Expected at least 1 space before "."; 0 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 46ms; Memory: 4MB
[root@b8a7f7560dfb html]# cat sniff_match.php 
<?php

$x = match (1) {
    1,2,3 => 'asdas',
    default => 5
};

$a = '1'. '2';
[root@b8a7f7560dfb html]# ./vendor/bin/phpcs --version
PHP_CodeSniffer version 4.0.1 (stable) by Squiz and PHPCSStandards
[root@b8a7f7560dfb html]# php --version
PHP 8.3.30 (cli) (built: Jan 13 2026 22:36:55) (NTS gcc aarch64)
Copyright (c) The PHP Group
Zend Engine v4.3.30, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.30, Copyright (c), by Zend Technologies
    with Xdebug v3.5.0, Copyright (c) 2002-2025, by Derick Rethans
```

**Comment by jrfnl** [comment]
@HMartusewiczVox Thanks for reporting this. The default standard is PSR12 and [PSR12 does not contain that rule](https://www.php-fig.org/psr/psr-12/).

What you are looking for is a ruleset to enforce the PER standard, which is not yet available. See issue #29 for more information.

In the mean time, you could consider adding the [`Universal.WhiteSpace.CommaSpacing`](https://github.com/PHPCSStandards/PHPCSExtra#universalwhitespacecommaspacing-wrench-bar_chart-books) sniff from PHPCSExtra to your own ruleset. I believe that should fix your issue.
